app/src/main/AndroidManifest.xml

```diff
@@ -35,8 +35,13 @@
     <activity android:name=".ShortcutReceiverActivity">
       <intent-filter>
         <action android:name="android.intent.action.SEND" />
+        <action android:name="android.intent.action.VIEW" />
```

Comment:
`ShortcutReceiverActivity` was mainly for handling app shortcuts create & sharing content from other apps. Perhaps you can create a new activity for handling the app actions?